### PR TITLE
Mitz/fix small mistakes

### PR DESCRIPTION
**Pull request by bufferings**
Please check this pull request after #20 is closed.

Translating the doc, I found some mistakes(? maybe).
So please could you take a look at them?

### CODE REVIEW

File: docs/tutorials/2.1/usingthymeleaf.md [hunk 1]
```diff
@@ -1199,7 +1199,7 @@ Let's use this new syntax. Meet the `th:href` attribute:
 <a href="details.html" th:href="@{/order/details(orderId=${o.id})}">view</a>
 
 <!-- Will produce '/gtvg/order/3/details' (plus rewriting) -->
-<a href="details.html" th:href="@{/order/{orderId}/details(orderId=${o.id})}">view</a>
```
Comment (danielfernandez) [inline docs/tutorials/2.1/usingthymeleaf.md]: This was actually correct. This kind of URL variable processing is a Spring feature, mirrored by thymeleaf here.

File: docs/tutorials/2.1/usingthymeleaf.md [hunk 2]
```diff
@@ -1210,7 +1210,7 @@ Some things to note here:
  * We are allowed to use expressions for URL parameters (as you can see in `orderId=${o.id}`).
    The required URL-encoding operations will also be automatically performed.
  * If several parameters are needed, these will be separated by commas like `@{/order/process(execId=${execId},execType='FAST')}`
- * Variable templates are also allowed in URL paths, like `@{/order/{orderId}/details(orderId=${orderId}}}`
```
Comment (danielfernandez) [inline docs/tutorials/2.1/usingthymeleaf.md]: As above, this use of variable templates is OK.

**Comment by danielfernandez** [comment]
Thanks. Indeed some of those were mistakes, but the URL variable templates were OK. So after merging your pull request I've created commit @602bcc03735b27e634e6cb25987ce3086329ac51 for reverting only the changes related to this.

I don't understand at all what I reverted in your Japanese doc :-), I simply reverted all commit chunks which included URL variable templates. So I'd need you to have a look at that commit and see if after my changes your docs still makes sense. Thanks!

File: docs/tutorials/2.1/usingthymeleaf.md (hunk 1 above)
Comment (bufferings) [inline docs/tutorials/2.1/usingthymeleaf.md]: Oh, I got it. Thanks for your explanation.

File: docs/tutorials/2.1/usingthymeleaf.md (hunk 2 above)
Comment (bufferings) [inline docs/tutorials/2.1/usingthymeleaf.md]: ```@{/order/{orderId}/details(orderId=${orderId}}}```

The closing ```}}}``` should be ```})}``` ? I mean, the second one is ```)```?

**Comment by bufferings** [comment]
OK. I will send a pull request for Japanese parts. Again thanks for your kindness.

File: docs/tutorials/2.1/usingthymeleaf.md (hunk 2 above)
Comment (danielfernandez) [inline docs/tutorials/2.1/usingthymeleaf.md]: You're right. It should be `})}` instead of `}}}`

Comment (bufferings) [inline docs/tutorials/2.1/usingthymeleaf.md]: Got it. Can I fix them?

Comment (danielfernandez) [inline docs/tutorials/2.1/usingthymeleaf.md]: Yes please.

**Pull request merged**